components/elm/src/data_types/ColumnDataType.F90

```diff
@@ -2090,6 +2090,16 @@ subroutine col_cs_init(this, begc, endc, carbon_type, ratio, c12_carbonstate_var
             avgflag='A', long_name='total column carbon, incl veg and cpool but excl product pools', &
             ptr_col=this%totcolc)
 
+       this%prod10c(begc:endc) = spval
+       call hist_addfld1d (fname='PROD10C', units='gC/m^2', &
+            avgflag='A', long_name='10-yr wood product C', &
+            ptr_col=this%prod10c, default='inactive')
+
+       this%prod100c(begc:endc) = spval
+       call hist_addfld1d (fname='PROD100C', units='gC/m^2', &
+            avgflag='A', long_name='100-yr wood product C', &
+            ptr_col=this%prod100c, default='inactive')
+
```

Comment:
@sshu88 was this a temporary change to make sure that you could read the `prod10c` and `prod100c` for when fates was running?

Reply:
These are two outputs I added to the history output file for checking the change of product pool size. So it is a temporary change. But the product pool size shall be a basic output in land surface model when considering anthropogenic disturbances.

Reply:
To clarify, does this need to be available when fates is running?  The current issue is that the same call is made just below behind a `.not. use_fates` check, which means this will get called twice and cause a build failure for non-fates runs.  If it's not necessary for fates right now, would you mind removing these lines?  Otherwise, the part that is behind the `.not. use_fates` check could be removed.

Reply:
I have removed the part below `.not. use_fates` check.